CHANGELOG

```diff
@@ -0,0 +1,22 @@
+## What's Changed
```

Comment:
Before that line we'll need a `#`-level heading that states the version that these changes are referring to.